rollup.config.ts

```diff
@@ -18,7 +18,7 @@ const plugins = [
 const input = "lib/index.ts";
 
 export default defineConfig([{
-    input,
+    input: "lib/index.umd.ts",
     output: {
       name: "jwt_decode",
```

Comment:
If we wanted to be really consistent we could rename this to `jwt-decode` to match the NPM package name, but I think backwards compatibility is likely more important than such a trivial change.